### PR TITLE
make skbio dependance optional

### PR DESCRIPTION
**Pull request by amnona**
Move skbio imports in transforming.py optional

### CODE REVIEW

File: calour/transforming.py [hunk 1]
```diff
@@ -302,6 +300,12 @@ def center_log_ratio(exp: Experiment, method=lambda matrix: matrix + 1, centrali
     skbio.stats.composition.clr
     skbio.stats.composition.centralize
     """
+    try:
+        from skbio.stats.composition import clr, centralize as skbio_centralize
+    except ModuleNotFoundError as err:
+        logger.error('scikit-bio must be installed in order to use this function')
+        raise err
```
Comment (RNAer) [inline calour/transforming.py:307]: why you need to try and capture exception?? you are raising the err anyway!

Comment (amnona) [reply calour/transforming.py:307]: In order to make better instructions on why it's failing - scikit-bio is not installed...
don't think it complicates too much but not critical - if you feel strongly about it i can delete...

Comment (RNAer) [reply calour/transforming.py:307]: `ModuleNotFoundError: No module named 'dafasdf'` looks very clear to me. no need for extra code

**Comment by amnona** [comment]
ready for merge :)

**Pull request merged**